Userland/Libraries/LibBigInteger/SignedBigInteger.h

```diff
@@ -107,6 +107,13 @@ class SignedBigInteger {
     SignedBigInteger multiplied_by(const SignedBigInteger& other) const;
     SignedDivisionResult divided_by(const SignedBigInteger& divisor) const;
 
+    inline SignedBigInteger opposite() const
```

Comment:
`negated_value`? `opposite` feels not quite correctly descriptive

Reply:
Looks like a bad translation, I place my trust in you!